src/DotNetProjectFile.Analyzers/Rule.cs

```diff
@@ -757,6 +757,14 @@ public static class Rule
        tags: ["resx", "resources", "invariant", "localization"],
        category: Category.Bug);
 
+    public static DiagnosticDescriptor EscapeXmlNodesResourceValues => New(
+      id: 2005,
+      title: "Escape XML nodes of resource values",
+      message: "Escape the XML node in '{0}'.",
+      description: "To ensure correct handling, XML nodes within resource values should be escaped.",
```

Comment:
Just for clarification, is it XML nodes or HTML nodes that need to be escaped?

Reply:
Most likely the XML in these data values are HTML, but technically, all XML inside should be escaped. Hence the choice of words.